src/Limiters/limiter.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    quasimonotone_limiter!(ρstar, Qstar, min_Q, max_Q, space, tol)
+
+This limiter is inspired by Guba et al Optimization-based limiters for the
+spectral element method", JCP 2014, doi: https://doi.org/10.1016/j.jcp.2014.02.029
```

Comment:
```suggestion
This limiter is inspired by Guba et al Optimization-based limiters for the
spectral element method", JCP 2014, doi: https://doi.org/10.1016/j.jcp.2014.02.029
```
you can add this to `doc/refs.bib`, and then use `[name](@cite)`